data/json/professions.json

```
@@ -578,7 +578,7 @@
     "type": "profession",
     "ident": "mechanic",
     "name": "Home Mechanic",
-    "description": "Although you never got your driver's license, you've always loved cars.  At least now you'll never be wanting for materials.",
+    "description": "Although you never got your driver's license, you've always loved working on cars.  You've kept hold of some handy tools for the job, and at least now you'll never want for materials.",
```

I think this sounds better. If you're interesting making changes past descriptions, I'd cut out the bit about never getting a drivers license and give some skill in driving.
```suggestion
    "description": "Although you never got your driver's license, you've always loved working on cars.  You've kept hold of some handy tools for the job, and at least now you'll never want for parts.",
```

---

The top gear magazine already does that.

---

This seems like a good idea to me, if it's allowed.  It's a little odd that they can build cars but have zero experience driving them, anyway. Something like 2 mechanics, 2 driving would be identical in terms of skill points.

---

Driving isn't a very useful skill, you don't need to worry about keeping the points on skills the same.